JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/core/dataset/schema/attribute/MultidimensionalAttribute2d.java

```diff
@@ -0,0 +1,112 @@
+package ai.libs.jaicore.ml.core.dataset.schema.attribute;
+
+import org.api4.java.ai.ml.core.dataset.schema.attribute.IAttributeValue;
+
+/**
+ * Attribute for 2dDouble Arrays
+ *
+ * @author Lukas Fehring
+ *
+ */
+public class MultidimensionalAttribute2d extends MultidimensionalAttribute<double[][]> {
+
+	private static final long serialVersionUID = -3300254871190010390L;
+	private int xsize;
+	private int ysize;
+
+	public int getXsize() {
+		return this.xsize;
+	}
+
+	public int getYsize() {
+		return this.ysize;
+	}
+
+	/**
+	 * Attribute Constructor
+	 *
+	 * @param name attribute name
```

Comment:
@param <parameterName> parameter description, independent on the name as own sentence